EAGLE_SIP/BeagleBone_Blue.sch

```diff
@@ -15939,6 +15939,17 @@ EAGLE CAD Libraries on element14&lt;/a&gt;</description>
 <part name="GPIO1" library="Seeed-Connector -2016" deviceset="GROVE-CONNECTOR-SMD(4+2P-2.0)" device=""/>
 <part name="SD_BOOT" library="SparkFun-Electromechanical" deviceset="SWITCH-MOMENTARY-2" device="SMD-1101NE" value="SWITCH-MOMENTARY-2SMD-1101NE"/>
 <part name="GND25" library="SparkFun-Aesthetics" deviceset="GND" device=""/>
+<part name="R26" library="Strawson_Eagle_Lib" deviceset="RESISTOR" device="0402-RES" value="10K,1%"/>
```

Comment:
They are in all the schematics we have seen and are part of the schematic checklist for AM335.  Aron should be able to test this given he has working boards.  Also, I could mod one of my boards to test this out if you want.